lib/tasks/scrape.rake

```diff
@@ -18,7 +18,7 @@ namespace :scrape do
     subject_groups.each do |_code, group|
       puts "Updating group #{group["code"]}"
       subject_group = SubjectGroup.find_or_initialize_by(code: group["code"])
-      subject_group.name = capitalize_name(group["name"])
+      subject_group.name = capitalize_name(correct_typos(group["name"]))
```

Comment:
```suggestion
      subject_group.name = cleanup(group["name"])
```

wdyt about creating a function like `cleanup` or something like that, that does both the capitalization and typo correction? (also maybe we can avoid doing the `split` twice)

Reply:
I like what you are thinking 👀 

Maybe something like `format_name` instead?

Reply:
👍  sounds good